lib/EntryPlugin.js

```diff
@@ -17,12 +17,12 @@ class EntryPlugin {
 	 *
 	 * @param {string} context context path
 	 * @param {string} entry entry path
-	 * @param {EntryOptions | string=} options entry options (passing a string is deprecated)
+	 * @param {EntryOptions} options entry options
 	 */
 	constructor(context, entry, options) {
 		this.context = context;
```

Comment:
Throw a error here if `typeof options !== "object"` to help migrating

Reply:
done